docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.4'
+services:
+  netdata:
+    image: titpetric/netdata
+    container_name: netdata
+    restart: always
```

Comment:
@SuperSandro2000  The only thing I'm worried about is this. Maybe we should comment it out or specify a note about the restart policy?
To be honest, I created this MR just for this option, but the behavior differs from the CLI version and it's not stated anywhere

Reply:
I would change it to ``restart: unless-stopped`` so that you can easily stop the container and it restarts when the machine restarts.
IIRC container started from the cli will never restart.

Reply:
https://docs.docker.com/config/containers/start-containers-automatically/#use-a-restart-policy seems like it's possible to implement in CLI. Should I add this parameter?

Reply:
Did not think about that. Maybe add a note about it to let the user decide what he prefers.

Reply:
How's that now?

Reply:
LGTM